attestation-service/attestation-service/src/verifier/tdx/claims.rs

```diff
@@ -275,4 +272,229 @@ mod tests {
 
         assert_json_eq!(expected, claims);
     }
+
+    #[test]
+    fn test_parse_kernel_parameters() {
```

Comment:
I like the detailed test cases very much, thanks for this! Could you use https://crates.io/crates/rstest crate to make it more rusty?

Reply:
@Xynnn007 - Branch updated.